.github/workflows/sketcher-builder.yml

```diff
@@ -55,53 +55,54 @@ jobs:
           version: ${{ fromJson(env.VERSIONS_JSON).emscripten }}
 
       - name: Configure Dependency CMake
-        if: matrix.build-output != 'wasm'
         run: |
-            cmake -B build/external/ -S external/ -G Ninja \
+            ${{ matrix.build-output == 'wasm' && 'emcmake ' || '' }}\
```

Comment:
same question here, can `${{ matrix.build-output == 'wasm' && 'emcmake ' || '' }}` be factored out into a variable